db/migrate/20221115215744_work_upload_type.rb

```diff
@@ -0,0 +1,22 @@
+class WorkUploadType < ActiveRecord::Migration[7.0]
+  def up
+    # new column to store which type of upload mechanism the user wants for their files
+    add_column :work_versions, :upload_type, :string, default: 'browser', null: false
```

Comment:
I'm wondering whether this migration is going to work on a db with a existing work versions? My suspicion is that you'll need to set `null: true` and then do the update below and then change it to `false`.

Have you tested this on a db with data?

Reply:
I tested it on localhost which had a few rows of data and it worked fine.  I believe the default value is what allows null to be false in the migration.  Perhaps I can deploy to qa as well as see what happens?

Reply:
Deployed to QA and migration ran cleanly with 385 work versions across 253 works (5 were globus type, the rest were file upload).  No upload_types were left as nil or empty.